views/check-your-answers.html

```diff
@@ -30,8 +30,6 @@ <h3 class="govuk-heading-m">Reason {{ loop.index }}</h3>
             <dd class="app-check-your-answers__answer">
               {{ reason.reason | capitalize }}
             </dd>
-            <dd class="app-check-your-answers__change">
```

Comment:
Why were these removed were they just not being used?

Reply:
Yeah, they were just empty tags. Figured I'd clear them out.

Reply:
ok